addons/mail/static/src/model/model_manager.js

```diff
@@ -885,14 +967,14 @@ class ModelManager {
             const [hash, currentFieldName, relatedFieldName] = dependent.split(
                 this.DEPENDENT_INNER_SEPARATOR
             );
-            const field = Model.fields[currentFieldName];
+            const currentRelatedField = Model.fields[currentFieldName];
```

Comment:
Name is super confusing, `currentFieldName` is not always from relational field, and `currentRelatedField` is problematic with `relatedFieldName`. For coherence, should be `currentField`, or change name of field names in parts of dependent (`fieldName1` and `fieldName2` are even less confusing).

Reply:
Ok for name with 1 and  2.